maptivist/src/main/webapp/script.js

```diff
@@ -176,17 +176,33 @@ function createInfowindow(position) {
         <p>counter: </p>
     </div>
     <div class="flag">
-        <button>Flag</button>
+        <button onclick = "flagFunction()">Flag</button>
     </div>
   </div>`;
 
+
   var infowindow = new google.maps.InfoWindow({
     content: contentString
   });
 
   return infowindow;  
 }
 
+function flagFunction() {
+  var flagString = 
+    `<div class="flag-window">
+		<h1>Enter issue here</h1>
+		<input type="text" placeholder="Problem" id="flag-problem" name="flag-problem" required></input>
+		<button type="button" class="btn cancel" onclick="InfoWindow.close()">Close</button>
+    </div>`;
+
+
+	var flagWindow = new google.maps.InfoWindow({
+    content: flagString
+  });
+
```

Comment:
I think I figured out the issue after doing some debugging. The flag-window is being draw, it's just appearing off the map. I think there are two ways to fix it.

1) Figure out how to pass the marker into this function, and then call:
`figureWindow.open(map, marker)`

Or 2) Figure out how to pass the coordinates of the marker into this function, and then call:

```
flagWindow.setPosition(coordinates);
flagWindow.open(map);
```

Or 3) a quick and dirty thing would be:
```
  flagWindow.setPosition(map.getCenter());
  flagWindow.open(map);
```
But then the info window isn't quite in the right place.